plonky2x/core/src/frontend/hash/blake2/curta.rs

```diff
@@ -379,4 +398,36 @@ mod tests {
         circuit.verify(&proof, &input, &output);
         circuit.test_default_serializers();
     }
+
+    #[test]
+    #[cfg_attr(feature = "ci", ignore)]
+    fn test_blake2b_curta_chunk_aligned() {
+        let _ = env_logger::builder().is_test(true).try_init();
+
+        let mut builder = CircuitBuilder::<L, D>::new();
+
+        let zeros = "0".repeat(30720);
+        let msgs = [zeros];
+
+        let msg_bytes = msgs.map(|x| hex::decode(x).unwrap());
+
+        const MSG_LEN: usize = 15360;
+        let msg_len_const = builder.constant::<U32Variable>(MSG_LEN as u32);
+        let msg_var = builder.read::<ArrayVariable<ByteVariable, MSG_LEN>>();
+
+        let calculated_digest = builder.curta_blake2b_variable(msg_var.as_slice(), msg_len_const);
+
+        let expected_digest = builder.constant::<Bytes32Variable>(bytes32!(
+            "467ec8e17ec259e15e915668cb1de2e6dc9982d50957da267fe9e472a4418527"
+        ));
+
+        builder.assert_is_equal(calculated_digest, expected_digest);
+
+        let circuit = builder.build();
+        let mut input = circuit.input();
+        input.write::<ArrayVariable<ByteVariable, MSG_LEN>>(msg_bytes[0].clone());
+        let (proof, output) = circuit.prove(&input);
+        circuit.verify(&proof, &input, &output);
+        circuit.test_default_serializers();
+    }
```

Comment:
Test cases should use the `blake2` crate to compute the expected hash. Also, modify test cases to be over smaller lengths (as then we can run this in CI).